ruff.toml

```diff
@@ -0,0 +1,20 @@
+exclude = [
```

Comment:
Would rather we have a `pyproject.toml` to start bringing configs together in 1 place I think.

Reply:
Agreed.